pyopenetl/operations.py

```diff
@@ -330,7 +361,7 @@ def seed_table(self, read_table: str, read_chunksize: int, write_table: str) ->
             if its == 1:
                 # write to (an optionally different) table name in Cloud SQL
                 print(f"--> Creating new table {write_table} in Cloud SQL")
-                self.create_table_from_dataframe(write_table, df)
+                self.create_table_from_dataframe(write_table, df, dtypes=self.get_postgres_dtypes_for_temp_table(read_table))
```

Comment:
this makes sense, but I'm thinking we might wanna change the method name from `self.get_postgres_dtypes_for_temp_table(read_table)` to something less specific to temp tables now that we're using it outside of the context of temp table -- maybe `get_postgres_table_schema()`? We'll need to update other places we call this method, obvs